src/Promenade.Web/Views/Podcasts/Episode.cshtml

```diff
@@ -9,10 +9,23 @@
                 @Localizer[Promenade.PodcastEpisodePublished, Model.PodcastItem.Episode, Model.PodcastItem.PublishDate.Value.ToLongDateString()]
             </em>
         </div>
-        <audio controls class="mb-2 w-100">
+        <audio id="player" controls class="mb-2 w-100">
             <source src="@Model.PodcastItem.MediaUrl" type="@Model.PodcastItem.MediaType" />
         </audio>
         <p>@Model.PodcastItem.Description</p>
         <div><em>@Localizer[Promenade.KeywordsItem, Model.PodcastItem.Keywords]</em></div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+@if (!string.IsNullOrWhiteSpace(Context.Items[Ocuda.Promenade.Controllers.ItemKey.GoogleAnalyticsTrackingCode] as string))
+{
+    <script>
+        var player = document.getElementById("player");
+        player.addEventListener("play", function () {
+            gtag('event', 'Podcast play', {
+                'event_category': '@Model.Podcast.Title',
```

Comment:
We should probably ensure that any `'` characters in the podcast title are escaped, I think this could be `@Model.Podcast.Title?.Replace("'", "\'")` but you should double-check me on this. You can test for sure by making a test podcast named something like `awful"podcast-name'alert("hello");` (if we even allow that)

Reply:
Resolved in commit f460e7d